### PR TITLE
chore: promote bacherfl, add craigpastro

### PR DESCRIPTION
**Pull request by toddbaert**
@bacherfl has contributed significantly to flagd, OFO, but more recently also the go-contribs (he implemented the in-process flagd provider). He's also done work on the go-sdk. @craigpastro helped design and update the fractional evaluation algorithm, added contextual data to flagd evaluations, and also has fixed some flagd bugs

This PR:

 * promotes @bacherfl to maintainer in cloud-native team
 * adds @bacherfl to maintainer in go-sdk team
 * adds @craigpastro to the cloud-native team as an approver

### CODE REVIEW

File: config/open-feature/sdk-golang/workgroup.yaml [hunk 1]
```diff
@@ -7,12 +7,13 @@ approvers:
...
   - AlexsJones
+  - bacherfl
+  - davejohnston
+  - james-milligan
   - Kavindu-Dodan
-  - thomaspoignant
   - skyerus
-  - james-milligan
+  - thomaspoignant
   - toddbaert
```
Comment (toddbaert) [inline config/open-feature/sdk-golang/workgroup.yaml:17]: alphabetized and added bacherfl

**Comment by toddbaert** [comment]
@craigpastro @bacherfl please let us know if you're interested in these [roles](https://github.com/open-feature/community/blob/main/CONTRIBUTOR_LADDER.md) by responding in this PR.

**Review by beeme1mr — APPROVED** [review]

File: config/open-feature/cloud-native/workgroup.yaml [hunk 2]
```diff
@@ -4,17 +4,19 @@ repos:
...
   - beeme1mr
-  - bacherfl
+  - craigpastro
+  - justinabrahms
   - odubajDT
-  - agardnerIT
+  - tcarrio
+  - thisthat
+  - thomaspoignant
+
```
Comment (toddbaert) [inline config/open-feature/cloud-native/workgroup.yaml:15]: alphabetized and added `craigpastro`

**Review by staceypotter — APPROVED** [review]
LGTM! 

Thanks @bacherfl @craigpastro !

**Review by thomaspoignant — APPROVED** [review]

**Comment by bacherfl** [comment]
Thanks @toddbaert - I am interested in this role!

**Pull request merged**